examples/repl.rs

```diff
@@ -21,6 +24,15 @@ mod example {
         disassembler: DisassemblySource,
     }
 
+    /// The subcommands that are acceptable for backtrace.
+    enum BtType {
```

Comment:
Nit - I think having a more explicit name would convey the intent better:
```suggestion
    enum BacktraceType {
```

Reply:
Makes sense.